immobilus/logic.py

```diff
@@ -353,6 +357,14 @@ def stop(self):
         TIME_TO_FREEZE = self.previous_time_to_freeze
         TZ_OFFSET = self.previous_tz_offset
 
+    def tick(self, delta=None):
```

Comment:
you may set a default value as timedelta(seconds=1) instead of None. This will remove the `if` below

Reply:
Some argue that it is a [bad practice](https://docs.python-guide.org/writing/gotchas/) but I can fix it if you want :)